src/meds_etl/simple_csv.py

```diff
@@ -0,0 +1,220 @@
+import argparse
+import json
+import multiprocessing
+import os
+import pickle
+import random
+import shutil
+import subprocess
+import tempfile
+
+import jsonschema
+import meds
+import polars as pl
+import pyarrow.parquet as pq
+
+import meds_etl
+
+
+def load_file(decompressed_dir, fname):
+    if fname.endswith(".gz"):
+        file = tempfile.NamedTemporaryFile(dir=decompressed_dir)
+        subprocess.run(["gunzip", "-c", fname], stdout=file)
+        return file
+    else:
+        return open(fname)
+
+
+def process_file(args):
+    event_file, num_shards, temp_dir, decompressed_dir, index = args
+    print("Working on ", event_file)
+
+    with load_file(decompressed_dir, event_file) as temp_f:
+        table = pl.read_csv_batched(temp_f.name, infer_schema_length=0, batch_size=1_000_000)
+
+        batch_index = 0
+
+        while True:
+            batch_index += 1
+
+            batch = table.next_batches(1)
+            if batch is None:
+                break
+
+            batch = batch[0]
+
+            batch = batch.lazy().rename({c: c.lower() for c in batch.columns})
+
+            patient_id = pl.col("patient_id").cast(pl.Int64)
+            time = pl.col("time")
+            
+            
+            time = pl.coalesce(
```

Comment:
I think for CSV reading (which we may want to separate from the flat ETL) we should just let users specify time strings. There is so much variability here, and it is an easy thing to take as an arg.

Reply:
Good idea. I'll fix this and then resolve this comment.